Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Projection/ProjectionIntegrityViolationDetector.php

```diff
@@ -271,67 +316,65 @@ public function referenceIntegrityIsProvided(): Result
     /**
      * This is provided by the database structure:
      * reference relations with the same source and same name must have distinct positions
-     * @inheritDoc
      */
     public function referencesAreDistinctlySorted(): Result
     {
+        // TODO implement
```

Comment:
hello world, but you just added the todo, seems like it wasnt implemented beforehand as well

Reply:
correct. Todos can be searched for. There should be no stubs without